### PR TITLE
Translate Ruby 3.1.1 released (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: #2818

Translation of Ruby 3.1.1 Released.
https://github.com/ruby/www.ruby-lang.org/blob/master/en/news/_posts/2022-02-18-ruby-3-1-1-released.md

Actual diff fa239c7d3a6be6b9fe569499537aaa5d8158ec29

This release has list of actual issues, and some of those title is hard to translate. In those case, I try to changes ticket title for better explanation. ;)

### CODE REVIEW

File: ko/news/_posts/2022-02-18-ruby-3-1-1-released.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+---
+
+Ruby 3.1.1이 릴리스되었습니다.
+
+3.1 안정 버전 시리즈의 첫 TEENY 버전입니다.
+
+* [error_highlight가 -e 옵션과 함께 동작하지 않음](https://bugs.ruby-lang.org/issues/18434)
+* [YJIT 이 메서드 기본 인수와 키워드 인수를 함께 사용할 때 잘못된 순서로 메서드 인수를 넘기는 문제 수정](https://bugs.ruby-lang.org/issues/18453)
+* [Warning#warn 메서드를 찾을 수 없을 때 세그멘테이션 폴트가 발생함](https://bugs.ruby-lang.org/issues/18458)
+* [Pathname이 디렉터리 글로빙에서 ".."를 반환하지 않음](https://bugs.ruby-lang.org/issues/18436)
```
Comment (yous) [inline ko/news/_posts/2022-02-18-ruby-3-1-1-released.md:17]:
```suggestion
* [Pathname이 디렉터리 glob 수행 시 ".."를 반환하지 않음](https://bugs.ruby-lang.org/issues/18436)
```

File: ko/news/_posts/2022-02-18-ruby-3-1-1-released.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+* [ruby --help의 --jit-max-cache 기본값 설명을 변경](https://bugs.ruby-lang.org/issues/18469)
+* [3.1.0-dev `include`를 호출하면 Module이 초기화 완료로 취급됨](https://bugs.ruby-lang.org/issues/18292)
+* [Optionparser의 튜토리얼 링크가 깨짐](https://bugs.ruby-lang.org/issues/18468)
+* [다른 스레드에서 Enumerator의 요소를 yield로 호출하면 세그멘테이션 폴트가 발생함](https://bugs.ruby-lang.org/issues/18475)
```
Comment (yous) [inline ko/news/_posts/2022-02-18-ruby-3-1-1-released.md:21]:
```suggestion
* [다른 스레드에서 Enumerator의 요소를 yield 하면 세그멘테이션 폴트가 발생함](https://bugs.ruby-lang.org/issues/18475)
```

File: ko/news/_posts/2022-02-18-ruby-3-1-1-released.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+* [`active_decorator`가 Ruby 3.1.0에서 세그멘테이션 폴트가 발생함](https://bugs.ruby-lang.org/issues/18489)
```
Comment (yous) [inline ko/news/_posts/2022-02-18-ruby-3-1-1-released.md:22]:
```suggestion
* [`active_decorator`가 Ruby 3.1.0에서 세그멘테이션 폴트를 일으킴](https://bugs.ruby-lang.org/issues/18489)
```

File: ko/news/_posts/2022-02-18-ruby-3-1-1-released.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+* [Fiber에서 Process.daemon을 호출하면 세그멘테이션 폴트가 발생함](https://bugs.ruby-lang.org/issues/18497)
+* [0 << (2\*\*40)는 NoMemoryError가 발생하지만 0 << (2\*\*80)는 0을 반환함](https://bugs.ruby-lang.org/issues/18517)
+* [IO read/write/wait 훅 버그 수정](https://bugs.ruby-lang.org/issues/18443)
+* [자기 자신을 별칭으로 지정할 때의 메모리 누수](https://bugs.ruby-lang.org/issues/18516)
+* [에러: 선언하지 않은 식별자 'MAP_ANONYMOUS' 사용](https://bugs.ruby-lang.org/issues/18556)
+* [\[버그\] RubyVM::InstructionSequence.load_from_binary에서 T_NONE 객체에 마킹을 시도](https://bugs.ruby-lang.org/issues/18501)
+* [require를 통해 던져진 throw_data의 예외 처리](https://bugs.ruby-lang.org/issues/18562)
+* [`IpAddr` 인스턴스의 `IpAddr#to_range` 호출을 수정](https://bugs.ruby-lang.org/issues/18570)
```
Comment (yous) [inline ko/news/_posts/2022-02-18-ruby-3-1-1-released.md:30]:
```suggestion
* [`IpAddr` 프로즌 인스턴스의 `IpAddr#to_range` 호출을 수정](https://bugs.ruby-lang.org/issues/18570)
```

**Review by yous — APPROVED** [review]

**Comment by riseshia** [comment]
Thanks!!!!!!! :bow:

**Pull request merged**